### DISCUSSION TITLE
Pedal transpose stuck on?

### DISCUSSION THREAD

**Discussion by Antony-B** [GrandOrgue/grandorgue#1995] [Help] [answered]
Somehow all notes on my pedalboard are suddenly being transposed down a half-step no matter what organ I load. The midi settings for the pedal show the transpose function to be set at 0, so I know that isn’t the cause. I have tried unplugging things, reloading, restarting, reinstalling, etc. and I have run out of ideas as to what the cause could possibly be. I’ve been using the same setup for several years and have never had this happen before. Any ideas on what the issue might be? Thanks for any help!

**Comment by midimusic** [comment, marked as answer]
The fault may be in your pedalboard.
Assuming that your pedalboard outputs midi then I suggest testing it with midi-ox to check that it is outputting its midi correctly (i.e without transposing).
csw900

**Reply by Antony-B** [reply]
Thanks for that suggestion. It seems that you are correct that it is a fault of the pedalboard midi signal. When I play the bottom C the program identifies it as being B0, C# being C1, etc.. Why would it start doing this all of a sudden after years of working just fine? Any idea of how to correct this issue?

**Reply by midimusic** [reply]
It is a very strange fault and I do not know what could cause it. Perhaps you should be grateful for the years of good service and get a new one!!! Alternatively it is possible that GO could transpose and correct the pedal notes.
csw900

**Reply by hnb2907** [reply]
Hi,

In GO MIDI configuration for the pedalboard, just offset the lowest/highest MIDI note numbers by 1 for each semitone offset.  That will get GO to play the correct note again.


Cheers,
Chris.

**Reply by rousseldenis** [reply]
I would say there is something on the midi interface connecting the pedalboard to computer.

What kind of pedalboard do you have?

**Reply by Antony-B** [reply]
> Hi,
> 
> In GO MIDI configuration for the pedalboard, just offset the lowest/highest MIDI note numbers by 1 for each semitone offset. That will get GO to play the correct note again.
> 
> Cheers, Chris.

This is what I have been doing since it happened. I think I’m just annoyed that I can’t figure out the issue. I’ll probably just forget trying to fix it for now since using the transpose setting in the midi configuration works just fine to remedy the situation. Thanks for the help!

**Reply by Antony-B** [reply]
> I would say there is something on the midi interface connecting the pedalboard to computer.
> 
> What kind of pedalboard do you have?

It’s a pedalboard I midified myself years ago using a kit from midi-hardware.com.

**Reply by rousseldenis** [reply]
Ok, a quick look in a controller manual says that the two lowest notes on the keyboard would affect transpose when controller is in edit mode. I think you should read the manual to find out how to reset that.
